Code/Framework/AzFramework/Platform/iOS/AzFramework/Archive/ArchiveVars_iOS.h

```diff
@@ -7,5 +7,3 @@
  */
 
 #pragma once
```

Comment:
Can we remove this entire file?

Reply:
Likely, I'm currently preparing a 'clean' version of this PR, will take a look in there :)